Assets/PatchKit Patcher/Scripts/PatchKitLogo.cs

```diff
@@ -0,0 +1,86 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using System.Collections;
+using UniRx;
+using PatchKit.Unity.Patcher.Debug;
+
+namespace PatchKit.Unity
+{
+
+    public class PatchKitLogo : MonoBehaviour
+    {
+        [SerializeField]
+        private string patchKitWebsiteUrl;
+
+        [SerializeField]
+        private Texture2D cursorTexture;
+
+        [SerializeField]
+        private Vector2 cursorHotspot;
+
+        [SerializeField]
+        private Image image;
+
+        [SerializeField]
+        private Button button;
```

Comment:
Please use `public` for Unity fields. I know that `[SerializeField]` is much better way, but we already set convention to `public` fields.